app/views/node.html

```diff
@@ -3,203 +3,285 @@
-            <p class="announcement-text">{{ 'NODE.POST_RECOVERY' | translate }}</p>
-          </div>
+        <div class="node-detail__subsection__cell--small">
+          <p class="cr-panel-block__item__header ng-binding">
+            15{{ 'NODE.MINUTES' | translate }}
+          </p>
+          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.load['1'] >= 0">
+            {{ node.load['15']|number:2 }}
+          </p>
         </div>
       </div>
+    </div>
 
-      <div class="row">
-        <div class="col-sm-12">
-          <h3 class="panel-headline">{{ 'NODE.DISK_OPERATIONS' | translate }}</h3>
-        </div>
-        <div class="col-sm-6 col-xs-6">
-          <div class="panel">
-              <p class="announcement-heading" ng-show="node.iostats.rps > -1">
-                <span class="iops-value">{{ node.iostats.rps|number:1 }}</span>
-                <span class="iops-value">{{ node.iostats.wps|number:1 }}</span>
-              </p>
-              <p class="announcement-heading" ng-hide="node.iostats.rps > -1">-</p>
-              <p class="announcement-text">
-                <span class="iops-label">{{ 'NODE.READ' | translate }} {{ 'NODE.OPS' | translate }}</span>
-                <span class="iops-label">{{ 'NODE.WRITE' | translate }} {{ 'NODE.OPS' | translate }}</span>
-              </p>
-          </div>
-        </div>
-        <div class="col-sm-6 col-xs-6">
-          <div class="panel">
-              <p class="announcement-heading" ng-show="node.iostats.rbps > -1">
-                <span class="iops-value">{{ node.iostats.rbps|bytes }}/s</span>
-                <span class="iops-value">{{ node.iostats.wbps|bytes }}/s</span>
-              </p>
-              <p class="announcement-heading" ng-hide="node.iostats.rbps > -1">-</p>
-              <p class="announcement-text">
-                <span class="iops-label">{{ 'NODE.READS' | translate }}</span>
-                <span class="iops-label">{{ 'NODE.WRITES' | translate }}</span>
-              </p>
-          </div>
+    <div class="node-detail__subsection--small">
+      <h2 class="">{{ 'NODE.CPU_CORES' | translate }}</h2>
+      <div class="node-detail__subsection__detail cr-panel-block">
+        <div class="node-detail__subsection__cell--small">
+          <p class="cr-panel-block__item__header ng-binding">
+            {{ 'NODE.CORES' | translate }}
+          </p>
+          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.num_cores > 0">
+            {{ node.num_cores }}
+          </p>
         </div>
       </div>
+    </div>
+
+    <div class="node-detail__subsection--medium">
+      <h2 class="">{{ 'NODE.SHARDS' | translate }}</h2>
+      <div class="node-detail__subsection__detail cr-panel-block">
+        <div class="node-detail__subsection__cell">
+          <p class="cr-panel-block__item__header ng-binding">
```

Comment:
Nothing, and im not even sure why I put this in here yesterday. I'll remove it...